src/sentry/static/sentry/app/components/group/inboxBadges/inboxReason.tsx

```diff
@@ -30,6 +31,46 @@ function InboxReason({inbox, fontSize = 'sm'}: Props) {
       ? `More than ${Math.round(count / EVENT_ROUND_LIMIT)}k`
       : `${count}`;
 
+  function getTooltipDescription({
+    until,
```

Comment:
you have access to the reason details above, so you can use it directly instead of passing it as a parameter

Reply:
Ah, that seems cleaner! Thanks :)